assets/src/js/homepage_transparent_nav.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Handles the transition between the new styles and default styles for the
+ * Navigation Menu when it is scrolled.
+ */
+export const setupTransparentNavHomepage = () => {
+  let isScrolled = false;
+
+  /**
+   * Executes when the DOM is fully loaded.
+   *
+   * @param {Function} fn - The callback function to run when the DOM is ready.
+   */
+  function ready(fn) {
```

Comment:
I don't think we need this function, since we already call `setupTransparentNavigation` [once when DOM is loaded](https://github.com/greenpeace/planet4-master-theme/blob/ee53957761d3412f3779c32bad3d8b4aca975e71/assets/src/js/app.js#L22)